src/components/ProjectPage/ProjectPage.js

```diff
@@ -43,20 +39,20 @@ class ProjectPage extends Component<Props> {
       !nextProps.project ||
       this.props.project.id !== nextProps.project.id
     ) {
-      this.loadNewProjectOrBail(nextProps.project);
+      this.loadProjectDependencies(nextProps.project);
     }
   }
 
-  loadNewProjectOrBail(project: Project) {
-    const { history, loadDependencyInfoFromDisk } = this.props;
+  loadProjectDependencies(project: Project) {
+    const { loadDependencyInfoFromDisk } = this.props;
 
     if (project) {
       loadDependencyInfoFromDisk(project.id, project.path);
     } else {
       // If the selected project was not successfully resolved, that means
       // it must have been deleted. We should redirect the user to the main
       // screen.
-      history.push('/');
+      // history.push('/');
```

Comment:
Does this still need to be re-implemented?

Reply:
Good catch! Meant to get rid of this whole method, and just call `loadDependencyInfoFromDisk` directly when needed